src/js/common/schemaform/helpers.js

```diff
@@ -488,3 +488,24 @@ export function expandArrayPages(pageList, data) {
 
   return result.currentList;
 }
+
+/**
+ * getPageKeys returns a list of keys for the currently active pages
+ *
+ * @param pages {Array} List of page configs
+ * @param formData {Object} Current form data
+ * @returns {Array} A list of strings with the page key from the page config
```

Comment:
There are some options for documenting anonymous objects ([name](http://usejsdoc.org/tags-name.html), [type](http://usejsdoc.org/tags-type.html), [closure compiler example](https://github.com/google/closure-compiler/wiki/Annotating-JavaScript-for-the-Closure-Compiler#record-type)), would any of these make sense here?